src/test/java/com/example/javawebcourse/api/annotation/EnabledFeatureToggle.java

```diff
@@ -0,0 +1,15 @@
+package com.example.javawebcourse.api.annotation;
+
+import com.example.javawebcourse.featuretoggle.FeatureToggles;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface EnabledFeatureToggle {
+
+    FeatureToggles value();
```

Comment:
```suggestion
    FeatureToggle[] value();
```

This would be the most suitable way to make annotaton more flexible, since you may want to enable multiple features at the same time